src/v2/Apps/Order/Routes/__tests__/Utils/OrderAppTestPage.tsx

```diff
@@ -114,6 +133,7 @@ export class OrderAppTestPage extends RootTestPage {
   }
 
   async setOfferAmount(amount: number) {
+    if (this.find(PriceOptions).length) this.selectCustomAmount()
```

Comment:
From now on the `OfferInput` is not visible until the user clicks on the custom option. So this first step is needed.